sql/7_0_0-to-7_0_1_upgrade.sql

```diff
@@ -105,3 +105,26 @@
 --    desc: Change Layout edit options.
 --    arguments: mode(add or remove) layout_form_id the_edit_option comma_separated_list_of_field_ids
 
+#IfNotTable questionnaire_repository
+CREATE TABLE `questionnaire_repository` (
+    `id` bigint(21) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `uuid` binary(16) DEFAULT NULL,
+    `questionnaire_id` varchar(255) DEFAULT NULL,
+    `provider` int(11) UNSIGNED DEFAULT NULL,
+    `version` int(11) NOT NULL DEFAULT 1,
+    `created_date` datetime DEFAULT current_timestamp(),
+    `modified_date` datetime DEFAULT current_timestamp(),
+    `name` varchar(255) DEFAULT NULL,
+    `type` varchar(63) NOT NULL DEFAULT 'Questionnaire',
+    `profile` varchar(255) DEFAULT NULL,
+    `active` tinyint(2) NOT NULL DEFAULT 1,
+    `status` varchar(31) DEFAULT NULL,
+    `source_url` text,
+    `code` varchar(255) DEFAULT NULL,
+    `code_display` text,
+    `questionnaire` longtext,
+    `form_js` longtext,
+    PRIMARY KEY (`id`),
```

Comment:
also need (here and in upgrade table):
```
UNIQUE KEY `uuid` (`uuid`)
```

Reply:
Why? Is a uuid and by definition is unique

Reply:
Will increase performance when seeking with uuid especially when uuid's take over id's in the distant future (in other tables have been doing this with uuid's).